cirq-core/cirq/vis/heatmap.py

```diff
@@ -299,6 +310,42 @@ def plot(
             ax = cast(plt.Axes, ax)
         original_config = copy.deepcopy(self._config)
         self.update_config(**kwargs)
+
+        highlighted_qubits = _as_set(kwargs.get("highlighted_qubits", ()))
+        if highlighted_qubits:
+            edgecolors = tuple(
+                (
+                    "red"
+                    if any(q in highlighted_qubits for q in qubits)
+                    else self._config["collection_options"].get("edgecolors", "grey")
+                )
+                for qubits in sorted(self._value_map.keys())
+            )
+            linestyles = tuple(
+                (
+                    "solid"
+                    if any(q in highlighted_qubits for q in qubits)
+                    else self._config["collection_options"].get("linestyles", "dashed")
+                )
+                for qubits in sorted(self._value_map.keys())
+            )
+            linewidths = tuple(
+                (
+                    4
+                    if any(q in highlighted_qubits for q in qubits)
+                    else self._config["collection_options"].get("linewidths", 2)
+                )
+                for qubits in sorted(self._value_map.keys())
+            )
+        else:
+            edgecolors = self._config["collection_options"].get("edgecolors", "grey")
+            linestyles = self._config["collection_options"].get("linestyles", "dashed")
+            linewidths = self._config["collection_options"].get("linewidths", 2)
```

Comment:
The main-branch version uses matplotlib defaults here which users may have set via [RcParams](https://matplotlib.org/stable/api/matplotlib_configuration_api.html#matplotlib.RcParams).
Let us keep that behavior, ie, please delete and move the `self._config` update on lines 345-348 under the `if highlighted_qubits` block.

Reply:
Yes, it was an oversight. Thanks. In that case, we don't even need the `else` branch, so I've deleted that as well.